Cargo.toml

```diff
@@ -21,3 +21,11 @@ members = [
     "discovery-handler-modules/udev-discovery-handler",
 ]
 resolver = "2"
+
+[workspace.package]
```

Comment:
nit: We may want to also set the authors key here for consistency ?

Reply:
I wasn't sure what to put and punted but i think this is worth unifying. I set the global author to be "The Akri Team"